docs/sources/clients/lambda-promtail/_index.md

```diff
@@ -95,6 +95,10 @@ For those using Cloudwatch and wishing to test out Loki in a low-risk way, this
 
 Note: Propagating logs from Cloudwatch to Loki means you'll still need to _pay_ for Cloudwatch.
 
+### WAF logs
```

Comment:
WAF?   Web Application Framework?  Write Amplification Factor?  There are 48 definitions for WAF on Acronym Finder.  I really had to hunt through the Amazon docs, but apparently it is "web application firewall".